src/tests/functional/plugin/shared/src/snippets/add.cpp

```diff
@@ -46,9 +45,8 @@ void Add::SetUp() {
     auto f = ov::test::snippets::AddFunction(inputDynamicShapes);
     function = f.getOriginal();
     setInferenceType(type);
-    if (!configuration.count(InferenceEngine::PluginConfigInternalParams::KEY_SNIPPETS_MODE)) {
-        configuration.insert({InferenceEngine::PluginConfigInternalParams::KEY_SNIPPETS_MODE,
-                              InferenceEngine::PluginConfigInternalParams::IGNORE_CALLBACK});
+    if (!configuration.count("SNIPPETS_MODE")) {
```

Comment:
I didn't use `ov::internal_cpu::snippets_mode` because to do so I need to add cpu plugin dependency via cmake

Reply:
interesting why snippets are part of shared tests? they are used in CPU only..

Reply:
@a-sidorova do you know is there some plan to make snippets as common component for several plugins? If no, may we move the tests from shared to cpu specific?

Reply:
@vurusovs `snippets` lib has common API that several plugins may use this component. At the moment only CPU Plugin uses the library and we don't have any plans to add snippets as component in other plugins in the near future. But despite this I'd suggest to leave it so as shared tests since `snippets` is common library and tests for common part should be shared to avoid reverse moving of tests to a shared folder in the future.